Paintroid/src/main/res/values/string.xml

```diff
@@ -21,6 +21,7 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
 
     <string name="app_name">Pocket Paint</string>
+    <string name="ext_storage_directory_name" translatable="false">Pocket Paint</string>
```

Comment:
move this string down to the <!-- extras --> section

btw, you can change your PR by simply pushing the same branch with an additional commit ;)
Also please squash your commits before pushing.